src/api/schema.ts

```diff
@@ -674,4 +674,14 @@ export const lists = {
     },
     access: allowAll,
   }),
+  Feedback: list({
+    fields: {
+      type: text(),
+      satisfaction: text(),
+      improvement: text(),
+      otherComments: text(),
+      referralUrl: text(),
```

Comment:
Should this be required? If so could it come from `select` field with options?

Reply:
but it could come from any page in the service?